js/samples/04.ai.d.chainedActions.listBot/src/prompts/summarize/skprompt.txt

```diff
@@ -53,7 +56,12 @@ Here's the users query:
 ```
 
 Use the template to create an adaptive card.
+Create a Container for each list.
+Replace the first TextBlock text in the Container with the List name.
+Replace the second TextBlock text in the Container with the list items.
 Take into account any changes asked for by the user.
-Always replace the placeholder text, List n and Item n, with real data.
+-----------------------------------------------------------------------
+
+User: summarize my lists
```

Comment:
What if the user wants to summarize only one list? The "user query" section is where the model is able to include the users' preference.